tls/s2n_tls_parameters.h

```diff
@@ -197,6 +197,10 @@
 #define S2N_TLS_MAXIMUM_RECORD_LENGTH   (S2N_TLS_MAXIMUM_FRAGMENT_LENGTH + S2N_TLS_RECORD_HEADER_LENGTH)
 #define S2N_TLS_MAX_FRAG_LEN_EXT_NONE   0
 
+/* TLS1.3 has a max fragment length of 2^14 + 1 */
+#define S2N_TLS13_MAXIMUM_FRAGMENT_LENGTH 16385
+#define S2N_TLS13_MAXIMUM_RECORD_LENGTH   (S2N_TLS13_MAXIMUM_FRAGMENT_LENGTH + S2N_TLS_RECORD_HEADER_LENGTH)
```

Comment:
if we are overallocating, should we allocate more to take into account possible AEAD expansion?

This limit
   is derived from the maximum TLSInnerPlaintext length of 2^14 octets +
   1 octet for ContentType + the maximum AEAD expansion of 255 octets
https://tools.ietf.org/html/rfc8446#page-77

Reply:
Good eye, i'll update the macros and include this.